src/MSAAdvanced.sol

```diff
@@ -87,6 +88,7 @@ contract MSAAdvanced is IMSA, ExecutionHelper, ModuleManager, HookManager {
         external
         payable
         onlyExecutorModule
+        withRegistry(msg.sender, MODULE_TYPE_EXECUTOR)
```

Comment:
this may not be needed as executor attestation would have been checked upon installation.

Reply:
this is more for ongoing security, eg an executor is installed and later a bug is found and the attestation revoked